source/Calamari.Shared/Deployment/Conventions/DependencyVariablesStrategies/GitDependencyVariablesStrategy.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using Calamari.Common.Plumbing.Variables;
+using Octopus.CoreUtilities.Extensions;
+
+namespace Calamari.Deployment.Conventions.DependencyVariablesStrategies
+{
+    public class GitDependencyVariablesStrategy : IDependencyVariablesStrategy
```

Comment:
This, and the interface/class below, allow us to provide different variables depending on the dependency type.